pkg/apis/ark/v1/pod_volume_backup.go

```diff
@@ -27,7 +27,7 @@ type PodVolumeBackupSpec struct {
 	Node string `json:"node"`
 
 	// Pod is a reference to the pod containing the volume to be backed up.
-	Pod corev1api.ObjectReference `json:"pod"`
+	Pod *corev1api.ObjectReference `json:"pod"`
```

Comment:
I don't think this is required. We ran into this problem when we were updating the generated code when we didn't have `k8s.io/api` available on the GOPATH. How did you regenerate things? The kubernetes-1.11.x tags for k8s.io/code-generator now allow us to run the generate groups shell script from anywhere (i.e. within ark), so we can remove the k8s.io/api bind mount (see https://github.com/heptio/ark/blob/32907931e13f38ef4e055652245eeb78d20ac76e/Makefile#L101-L102).

Reply:
Yeah! So, `hack/update-generated-crd-code.sh` calls `$GOPATH/src/k8s.io/code-generator/generate-groups.sh` which also requires k8s.io/apimachinery to be in the GOPATH as well. This is described in this issue: https://github.com/kubernetes/code-generator/issues/21

I have some other projects that use code generation and I add a `required` stanza to my Gopkg.toml for the k8s.io/code-generator repo so that we're not relying on out-of-tree code. I tried to set this up for Ark but that would be complicated for y'all because you prune non-go files, which prunes out the scripts.

When I did run the generator, however, both code-generator and apimachinery were checked out to the release-1.11 branch.